### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by ryanlow64**

### CODE REVIEW

**Review by chenxu20 — COMMENTED** [review]
LGTM in general!

File: src/test/java/seedu/address/logic/commands/event/AddEventCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.ModelStub;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.client.Client;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventType;
+import seedu.address.model.event.Note;
+import seedu.address.model.property.Property;
+
+class AddEventCommandTest extends AddCommandTest<Event> {
```
Comment (chenxu20) [inline src/test/java/seedu/address/logic/commands/event/AddEventCommandTest.java:32]: Consider adding a test case for duplicate event scenario, i.e. same client, property, eventType, time but different notes.

**Review by chenxu20 — COMMENTED** [review]
LGTM in general!

File: src/main/java/seedu/address/logic/commands/event/AddEventCommand.java [hunk 2]
```diff
@@ -45,9 +45,10 @@ public class AddEventCommand extends Command {
             .toString();
 
     public static final String MESSAGE_SUCCESS = "New event added: %s";
-    public static final String MESSAGE_DUPLICATE_EVENT = "This event already exists in the address book.";
+    public static final String MESSAGE_EVENT_CONFLICT = "Event conflicts with existing event.";
     public static final String MESSAGE_INVALID_PROPERTY_ID = "Invalid property ID.";
     public static final String MESSAGE_INVALID_CLIENT_ID = "Invalid client ID.";
+    public static final String MESSAGE_EVENT_IN_PAST = "Event cannot be before 01-01-2025 0000.";
```
Comment (mjyan02) [inline src/main/java/seedu/address/logic/commands/event/AddEventCommand.java:51]: Everything is good.

File: src/test/java/seedu/address/logic/parser/event/AddEventCommandParserTest.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.event.AddEventCommand;
+import seedu.address.logic.parser.AddCommandParserTest;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventType;
+import seedu.address.model.event.Note;
+
+class AddEventCommandParserTest extends AddCommandParserTest<Event> {
```
Comment (chenxu20) [inline src/test/java/seedu/address/logic/parser/event/AddEventCommandParserTest.java:25]: Very comprehensive test!

**Review by chenxu20 — APPROVED** [review]
LGTM
